src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -50,9 +50,9 @@
     <Reference Include="FontAwesome.WPF">
       <HintPath>..\..\extern\FontAwesome\FontAwesome.WPF.dll</HintPath>
     </Reference>
-    <Reference Include="Greg, Version=1.0.5892.25330, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\extern\greg\Greg.dll</HintPath>
+    <Reference Include="Greg, Version=1.0.5950.12824, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\Greg.1.0.5950.12824\lib\net45\Greg.dll</HintPath>
+      <Private>True</Private>
```

Comment:
CopyLocal should generally be False, unless we really need to copy it to the output directory during the build. There are some dlls such as DynamoServices should never be referenced with "CopyLocal=True", it creates multiple copies of this dll and has -ve side effect.